enterprise/backend/src/metabase_enterprise/advanced_config/file.clj

```diff
@@ -108,7 +108,7 @@
    [metabase.util.files :as u.files]
    [metabase.util.i18n :refer [trs tru]]
    [metabase.util.log :as log]
-   [yaml.core :as yaml]))
+   [metabase.util.yaml :as yaml]))
```

Comment:
All namespaces should use `metabase.util.yaml` instead of the YAML lib directly where possible.

Reply:
We can add it to the kondo `:discouraged-namespace` list to enforce that.